runtime/src/lib.rs

```diff
@@ -431,6 +431,10 @@ impl pallet_accounts::EthFragContract for Runtime {
 	}
 }
 
+parameter_types! {
+	pub const TicketsAssetId: u64 = 1337;
```

Comment:
This seems duplicated in some places, can they be moved to a common location?

Reply:
This would actually be that common location :) in fact, it is where all the other Config in the other pallets read the value from.
I will check again for duplicates in tests maybe..

Reply:
Yeah saw 2 other usages in tests I think, probably reference this value from there.

Reply:
I will refactor tests with another PR since it is lower priority and it would require some work that may not be so trivial.